content/2022-06-03-brigade-2.6.0-and-friends.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Brigade 2.6.0 and Friends"
+slug: "2022-06-03-brigade-2.6.0-and-friends"
+description: "Brigade 2.6.0 has been released"
+date: "2022-06-03"
+author: "Kent Rancourt"
+summary: "Brigade 2.6.0 and several new peripherals have been released!"
+---
+
+Hello, Brigadiers! We hope everyone's had a great week! It's certainly been a
+great one for us because we're closing it out with the release of Brigade v2.6.0
+and a few new peripherals!
+
+Brigade v2.6.0 continues the trend of small, frequent releases that are more
+evolutionary than revolutionary. Apart from the bug fixes, doc improvements, and
+other maintenance that go into every release, there's just one noteworthy change
+in v2.6.0 -- Brigade is now allowing longer label and qualifier values on events
+and in projects' event subscriptions. As small a change as this may seem, it's a
+key enabler for an important new peripheral...
+
+Today, we're announcing that the
+[Brigade CloudEvents Gateway](https://github.com/brigadecore/brigade-cloudevents-gateway)
+has reached general availability with its v1.0.0 release. Support for
+CloudEvents is _monumentally_ important because of its broadly useful potential
+for integrating Brigade with innumerable third-party services that also act as
+CloudEvent producers.
+
+To illustrate this new gateway's utility, let's consider Microsoft Azure.
+[Azure Event Grid](https://docs.microsoft.com/en-us/azure/event-grid/overview)
+can subscribe to events originating from many, if not most, types of Azure
+resources, transform them to CloudEvents, and deliver those over HTTPS to any
+CloudEvents consumer -- such as the Brigade CloudEvents gateway. And just like
+that -- Brigade v2.6.0+ scripts can be triggered by just about anything in
+Azure. And Azure is just the tip of the iceberg since the CloudEvents 1.0
+specification is being so widely adopted throughout the cloud-native community.
+
+And that's not all.
+
+Within the past two weeks, we've also cut the first Brigade v2-compatible
+[Brigade Bitbucket Gateway](https://github.com/brigadecore/brigade-bitbucket-gateway)
+release and the
+[Brigade Noisy Neighbor](https://github.com/brigadecore/brigade-noisy-neighbor)
+has reached general availability with its v1.0.0 release.
+
+The Bitbucket gateway is admittedly not as full-featured as the
+[Brigade GitHub Gateway](https://github.com/brigadecore/brigade-github-gateway),
+but provides Brigade v2 with Bitbucket integration on par with that which had
+been available for Brigade v1, which is important in light of Brigade v1 having
+reached end-of-life on June 1.
```

Comment:
I don't think you have to say this like it's a bad thing. how about something more along the lines of saying the Bitbucket gateway is younger than the GitHub gateway and brings that VCS up to par with v1.

Reply:
Whoops... I didn't complete my thought here. After mentioning parity with v1, I meant to go on to say that we would happily work with the community if anyone wants to help bring this gateway on par with what we have for GitHub. I'll work that in there.

Reply:
> The Bitbucket gateway provides Brigade v2 with Bitbucket integration on par with
that which had been available for Brigade v1, which is important in light of
Brigade v1 having reached end-of-life on June 1. If the community expresses
interest in it, we'd be eager to partner with the community contributors to
expand this gateway to support
[Bitbucket Cloud Apps](https://support.atlassian.com/bitbucket-cloud/docs/bitbucket-cloud-apps-overview/),
which would put its functionality on par with that offered by the
[Brigade GitHub Gateway](https://github.com/brigadecore/brigade-github-gateway).